migration/v0.20.x-v0.21.x/upgrade-apps.md

```diff
@@ -4,11 +4,19 @@
 
 ## Steps
 
-1. Remove grafana-ops chart with all the dashboards: `migration/v0.20.x-v0.21.x/remove-grafana-ops.sh`
+1. Update apps configuration:
 
-2. Rollover all streams to make fluentd have all logging fields/mappings: `migration/v0.20.x-v0.21.x/data-stream-rollovers.sh`
+    This will take a backup into `backups/` before modifying any files.
 
-3. Upgrade applications:
+    ```bash
+    bin/ck8s init
+    ```
+
+2. Remove grafana-ops chart with all the dashboards: `migration/v0.20.x-v0.21.x/remove-grafana-ops.sh`
+
+3. Rollover all streams to make fluentd have all logging fields/mappings: `migration/v0.20.x-v0.21.x/data-stream-rollovers.sh`
```

Comment:
```suggestion
@robinelastisys why is this nessecary? If nessecary why is it done before apps apply?
```

Reply:
It is necessary as fluentd needs to have all fields mapped out in OpenSearch to have it work correctly. If not, fluentd will say it simply cannot map the log field.

Reply:
Okay, I just fail to see what the impact of rolling over before upgrading has, the same mappings will be present after the rollover when fluentd pushes logs to opensearch.

Reply:
@Pavan-Gunda or @viktor-f can you try an upgrade without running this?

Reply:
I will do that!!